vagrant/pythonstartup.py

```diff
@@ -19,5 +19,7 @@ def save_history():
 try:
     import sideboard
     from uber.common import *
```

Comment:
This first line lets us just do session stuff directly in the REPL without needing to do ``with Session() as session`` first.  Which you should never, ever do in your production code, but is fine at the REPL.  (Also you can do it in production code if you REALLY know what you're doing, but even then it's not recommended.)